src/components/button.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function Button({ value, buttonClick, valueType }) {
+  const onClick = () => {
+    buttonClick(value);
+  };
+
+  const styles = 'w-[81px] h-[45px] p-2.5 rounded-[5px] border border-black flex-col justify-center items-center gap-2.5 inline-flex';
+  const backgroundColor = valueType === 'operand' ? styles.concat(' bg-stone-50') : styles.concat(' bg-stone-300');
```

Comment:
It is ok. But, you could simplify the style definition doing something like this:
```
const styles = `w-[81px] h-[45px] p-2.5 rounded-[5px] border border-black flex-col justify-center items-center gap-2.5 inline-flex ${valueType === 'operand' ? 'bg-stone-50' : 'bg-stone-300'} `
```